### DISCUSSION TITLE
Copilot login from MS visual code failed reason in corporate network

### DISCUSSION THREAD

**Discussion by Rajashekar-Bommideni** [community/community#80886] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi folks,

I started using github copilot  for VS code IDE recently.
I not able sign in to from my office network.
it display this error **[ERROR] [default] [2023-12-19T08:46:42.070Z] Extension activation failed: "connect ETIMEDOUT 140.82.113.6:443"**
can you help me what I am missing to configure.

**Comment by dufve** [comment]
It appears to be a network timeout issue where VS Code is unable to connect to GitHub's servers. Here are a few quick checks:

1. Confirm stable internet connectivity.
2. Check with IT regarding network or firewall restrictions.
3. Ensure your VS Code is configured correctly for any office proxy settings.

If the problem persists, reaching out to your IT department for further assistance would be advisable.

**Reply by Rajashekar-Bommideni** [reply]
Thank you for the response.
Can you guide me or share the document, where I can follow the steps to configure proxy settings

**Reply by dufve** [reply]
You can find some documentation for proxy configuration [here](https://code.visualstudio.com/docs/setup/network#_proxy-server-support).

However I find it more probable that there are some Firewall configurations/network settings set in place from your IT-department. That would mean that you can't resolve the problem yourself, and you'd need to contact them.

I would advise consulting your IT-department before making major changes to your proxy settings, as if they are there they are probably there for a reason. Although improbable, they might even have blocked the GitHub Copilot API on a global network level. There's really no other way to know than to contact them.

**Reply by Rajashekar-Bommideni** [reply]
Thanks for your suggestion.
Will proceed with the same. :)
